### DISCUSSION TITLE
NoModificationAllowedError

### DISCUSSION THREAD

**Discussion by mikebronner** [livewire/livewire#4334] [7. Bugs (archived)]
We're seeing quite a few of these errors:

```
NoModificationAllowedError Failed to set the 'outerHTML' property on 'Element': This element has no parent node. 
    ../js/connection/index.js:141:14 Connection.value
    ../js/connection/index.js:132:38 HTMLDivElement.<anonymous>
```

We're not quite sure what exactly is triggering this, but it seems it might be related to when Livewire shows 500 errors.

**Comment by joshhanley** [comment]
@mikebronner that seems odd, hard to try and debug it though if we can't replicate what is causing it. If you do find out what triggers it so we can replicate it, we can look into trying to get it fixed 🙂

**Comment by alejandrotrevi** [comment]
I'm also having this error, it seems like it's related to this piece of code

https://github.com/livewire/livewire/blob/020ad095cf1239138b097d22b584e2701ec3edfb/js/connection/index.js#L151-L154

Error: NoModificationAllowedError: Failed to set the 'outerHTML' property on 'Element': This element has no parent node.
  at ? (../js/connection/index.js:152:15)

**Reply by joshhanley** [reply]
@alejandrotrevi can you share a component class and view that replicates the issue so we can test it? Thanks!

**Reply by alejandrotrevi** [reply]
Just found out what causes this issue, we have a notifications component this component pulls every 10s

When we deploy (we are not using envoyer) the requests to the server fail (because ofc the server is down) which causes a 503 http error which causes livewire to show an error modal and this js exception

Main issue in this case is not using envoyer or something similar but still I have no idea if the js exception is normal I will test a little bit more

**Reply by joshhanley** [reply]
@alejandrotrevi ah interesting! Yeah Livewire should handle the 503 and display it in a modal, but it shouldn't be triggering an exception.

Yeah if you can test some more and if you can consistently replicate it locally, then let us know how to replicate so we can investigate it.

Thanks!
